test/sample-test/Dockerfile

```diff
@@ -21,12 +21,9 @@ RUN cd /sdk/python && python3 setup.py install
 
 COPY ./test/sample-test /python/src/github.com/kubeflow/pipelines/test/sample-test
 COPY ./samples /python/src/github.com/kubeflow/pipelines/samples
-RUN cd /python/src/github.com/kubeflow/pipelines
 
 # Install argo.
-RUN echo "install argo"
-RUN mkdir -p /usr/local/bin/
-RUN ARGO_VERSION=v2.3.0 && curl -sSL -o /usr/local/bin/argo \
+RUN ARGO_VERSION=v2.3.0 && mkdir -p /usr/local/bin/ && curl -sSL -o /usr/local/bin/argo \
     "https://github.com/argoproj/argo/releases/download/$ARGO_VERSION/argo-linux-amd64" && \
     chmod +x /usr/local/bin/argo
 ENV PATH $PATH:/usr/local/bin/argo
```

Comment:
I'm not sure this line works - argo is not a directory

Reply:
I'm assuming that this line basically downloads a binary from the girhub repo and make it executable. -o does not require the parameter to be a dir. Please correct me if I'm wrong thanks!